samples/samples_utils.c

```diff
@@ -0,0 +1,215 @@
+#include <stdio.h>
+#include <string.h>
+#include <assert.h>
+#include <sys/platform/ppc.h>
+#include "nxu.h"
+#include "nx_dht.h"
+#include "nx_dbg.h"
+#include "nx_zlib.h"
```

Comment:
Do we really need these internal headers?

Reply:
There is a lot of `nx_dht.h` structures used by samples, and the nx state from `nx_zlib.h` is necessary to get access to the device handler.
I removed `nxu.h` and `nx_dbg.h` from this file.

Reply:
OK. What if we add a comment here with a TODO saying: "Stop using the following nx headers"?